addons/kiam/3.x/kiam-1.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kiam
+  labels:
+    kubeaddons.mesosphere.io/name: kiam
+    kubeaddons.mesosphere.io/provides: iam
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.5-1"
```

Comment:
Addon machinery currently only supports revisions and versions that are [semver](https://semver.org/) compliant.

I see upstream that the project doesn't use semver, we will need to come up with a semver compliant scheme for this Addon in order for it to be functional.

Whatever we end up doing, we should document that in comments here so future readers understand the context.